straxen/plugins/led_cal/led_calibration.py

```diff
@@ -151,10 +157,14 @@ def compute(self, raw_records):
 
         area = get_area(records, led_windows, self.area_averaging_length, self.area_averaging_step)
         temp["area"] = area["area"]
+
+        temp["triggered"] = triggered
+        temp["hit_position"] = led_windows[:, 0] - self.led_hit_extension[0]
+        temp["integration_window"] = led_windows
         return temp
 
 
-def get_records(raw_records, baseline_window, LED_cal_record_length):
+def get_records(raw_records, baseline_window, led_cal_record_length):
     """Determine baseline as the average of the first baseline_samples of each pulse.
 
     Subtract the pulse float(data) from baseline.
```

Comment:
This was also like this before, I think the reasoning was, that in case there have to be changes made to this internal datatype. But maybe @GiovanniVolta can comment on that.

Reply:
The fact that it was there before does not mean that it is optimal.

Reply:
I clarified with Giovanni, it is actually only for historical reasons, because this plugin existed before the other code. I will look if we can change this easily.

Reply:
Actually it is a bit more involved to change this. Due to the different handling of the baseline and the fact that the `data` field is an integer in the strax records and a float here. 
We are also here in a different, externally triggered mode. Giovanni and I think it's justifiable to keep it as is.

Reply:
Yes, I understand that it is not pleasant that led_calibration has his get_records, but since the DAQ setting and the purpose are different WRT to the default strax get_records (which, BTW, it is basically copied and pasted here) I think it is okay to leave it as it is